besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -638,6 +642,20 @@ void setBannedNodeIds(final List<String> values) {
           "Enable to accept clients certificate signed by a valid CA for client authentication (default: ${DEFAULT-VALUE})")
   private final Boolean isRpcHttpTlsCAClientsEnabled = false;
 
+  @Option(
+      names = {"--rpc-http-tls-protocol", "--rpc-http-tls-protocols"},
+      description = "Comma separated list of TLS protocols to support (default: TLSv1.2)",
```

Comment:
TLS 1.3 should be (at least part of) the default. It gives better performance and is more secure and should be well supported by now. I guess first preference should be 1.3 and if we are worried about 1.3 support we could add 1.2.

Reply:
You are right, updated.